docs/topics/returns.md

```diff
@@ -39,32 +39,18 @@ loop@ for (i in 1..100) {
 A `break` qualified with a label jumps to the execution point right after the loop marked with that label.
 A `continue` proceeds to the next iteration of that loop.
 
+> In some cases, you can apply `break` and `continue` *non-locally* without explicitly defining labels.
+> Such non-local usages are valid in lambda expressions used in enclosing [inline functions](inline-functions.md#break-and-continue).
+>
+{style="tip"}
+
 ## Return to labels
 
 In Kotlin, functions can be nested using function literals, local functions, and object expressions.
-Qualified `return`s allow us to return from an outer function.
-The most important use case is returning from a lambda expression. Recall that when we write the following,
-the `return`-expression returns from the nearest enclosing function - `foo`:
+Qualified `return`s allow you to return from an outer function.
```

Comment:
The `return`s looks a bit weird in terms of formatting. WDYT about making it singular?
"A qualified `return` allows you to return from an outer function."